### ISSUE TITLE
documentation: Validating nested arrays after creation with class-transformer

### ISSUE THREAD

**Issue by Sauermann** [typestack/class-validator#868]
### Description
In the README.md section about validating arrays containing nested objects, the requirements for validating objects created with [class-transformer](https://github.com/pleerock/class-transformer) are not made clear. This issue regards the following section of the documentation:
https://github.com/typestack/class-validator/blob/9f0c3e9eb6174ca5b5d00363adf108427ab3ef41/README.md#L332-L342

The problem is, that after creating a nested array with class-transformer with the above code as template, the objects within the array do not contain the class information.

### Proposed solution
I suggest to include the following additional information:

When validating nested objects created with [class-transformer](https://github.com/pleerock/class-transformer), it is required to specify the type of the nested objects as described in the [class-transformer documentation](https://github.com/typestack/class-transformer#working-with-nested-objects).

**Comment by vlapo** [comment]
I agree we can add small note about this information.

**Comment by gilsoncav** [comment]
I also agree. Stumbled here with the same misleading assumption. It worths a note.
